exonum-java-binding/launcher-plugin/README.md

```diff
@@ -0,0 +1,45 @@
+# Java Runtime Plugin for Exonum Launcher
+
+## Installation
+
+#### Requirements
+
+- [Python3](https://www.python.org/downloads/)
+- [`exonum-python-client`](https://github.com/exonum/exonum-python-client)
+- [`exonum-launcher`](https://github.com/exonum/exonum-launcher)
+
+```bash
+cd exonum-java-binding
+python3 -m pip install -e launcher-plugin
+```
+
```

Comment:
I'd mention that sources are generated during usual build of the core module, and can be explicitly generated with `mvn generate-sources -pl core` (e.g., after switching to a new revision).